test/fixtures/output/iam/rbac/role-binding/list-help-onprem.golden

```diff
@@ -1,41 +1,41 @@
-List the role bindings for a particular principal and/or role, and a particular scope.
+List role bindings assigned to a principal or role based on scopes.
 
 Usage:
   confluent iam rbac role-binding list [flags]
 
 Examples:
 Only use the `--resource` flag when specifying a `--role` with no `--principal` specified. If specifying a `--principal`, then the `--resource` flag is ignored. To list role bindings for a specific role on an identified resource:
 
-  $ confluent iam rbac role-binding list --kafka-cluster $KAFKA_CLUSTER_ID --role DeveloperRead --resource Topic
+  $ confluent iam rbac role-binding list --kafka-cluster lkc-o3hw2k --role DeveloperRead --resource Topic:my-topic
 
 List the role bindings for a specific principal:
 
-  $ confluent iam rbac role-binding list --kafka-cluster $KAFKA_CLUSTER_ID --principal User:my-user
+  $ confluent iam rbac role-binding list --kafka-cluster lkc-o3hw2k --principal User:my-user
```

Comment:
This doesn't look right. These examples are for the on-prem version of the command, but we're using a Cloud resource ID here.

Reply:
Changed to an example from the CP docs (`LRx92c9yQ+ws786HYosuBn`). Do you have a preferred generic example?

Reply:
In a few of our examples, we just pad these IDs with 0s. There's no established format though...
```suggestion
  $ confluent iam rbac role-binding list --kafka-cluster 0000000000000000000000 --principal User:my-user
```

Reply:
Changed to  0000000000000000000000. Not sure I like that, but better to be consistent for now.